tests/rt.sh

```diff
@@ -218,7 +218,7 @@ elif [[ $MACHINE_ID = orion.* ]]; then
 
   module use $PATHTR/modulefiles/${MACHINE_ID}
   module load fv3
-  module load gcc/8.3.0
+  #module load gcc/8.3.0
```

Comment:
Does this gcc change make the stochastic physics compile go through? I thought this commit will go with the makefile change in stochastic physics?